File: SoftUni Lessons/Python Development/Python Basics April 2019/Lessons and Problems/05 - Conditional Statements Exercise/07. World Swimming Record .py
import math

record = float(input())
distance_meters = float(input())
time_seconds = float(input())

real_time = distance_meters * time_seconds
added_time = math.floor(distance_meters / 15) * 12.5
total_time = real_time + added_time

if total_time < record:
    print(f"Yes, he succeeded! The new world record is {total_time:.2f} seconds.")
else:
    print(f"No, he failed! He was {abs(record - total_time):.2f} seconds slower.")

File: SoftUni Lessons/JavaScript Development/JavaScript Basics March 2020/08. УПРАЖНЕНИЕ ПРОВЕРКИ/07. World Swimming Record.js
function worldSwimmingRecord(input) {
    let record = Number(input.shift());
    let meters = Number(input.shift());
    let timeFor1M = Number(input.shift());

    let totalTime = (meters * timeFor1M) + (Math.floor(meters / 15)) * 12.5;

    if (totalTime < record) {
        console.log(`Yes, he succeeded! The new world record is ${totalTime.toFixed(2)} seconds.`);
    } else {
        console.log(`No, he failed! He was ${(totalTime - record).toFixed(2)} seconds slower.`);
    }
}

worldSwimmingRecord(["10464", "1500", "20"]);
// Expected output:
// No, he failed! He was 20786.00 seconds slower.

worldSwimmingRecord(["55555.67", "3017", "5.03"]);
// Expected output:
// Yes, he succeeded! The new world record is 17688.01 seconds.
